cmlreaders/path_finder.py

```diff
@@ -121,12 +121,12 @@ def _lookup_file(self, data_type):
         """ This will handle the special cases before passing the data through
             to _find_single_path
         """
-        subject_localization = self.subject
+        subject_montage = self.subject
 
-        # Some files/locations append the localization number, so to abstract
-        # that away from the user, we handle this internally
-        if self.localization != '0':
-            subject_localization = "_".join([self.subject, self.localization])
+        # Some files/locations append the montage number to the subject ID, so
+        # to abstract that away from the user, we handle this internally
+        if self.montage != '0':
+            subject_montage = "_".join([self.subject, self.montage])
```

Comment:
Is this right? I was under the impression that `R1XYZQ_1` referred to localization 1.

Reply:
As was I until I was looking at the flattened data index yesterday and it became clear that the value after the underscore always corresponds to the montage number. I confirmed verbally with @leondavis1, but we can confirm at the standup.

Reply:
Yes, this is correct